ts/Dashboards/EditMode/SidebarPopup.ts

```diff
@@ -436,8 +436,10 @@ class SidebarPopup extends BaseForm {
                                 'layoutChanged',
                                 (e): void => {
                                     if (newCell && e.type === 'newComponent') {
+                                        const chart =
+                                            newCell.mountedComponent.chart;
 
-                                        if (newCell.mountedComponent.chart) {
+                                        if (chart?.isDirtyBox) {
                                             const unbind = addEvent(
                                                 newCell.mountedComponent.chart,
```

Comment:
I know it's not a part of the changed code, but since we assign this to a variable, can we also change it here?